spec/System/HapistranoSpec.hs

```diff
@@ -40,7 +40,7 @@ genSourceRepo path = do
         , "echo testing > " ++ joinPath [clonePath, "README"]
         , "cd " ++ clonePath ++ " && git add -A"
         , "cd " ++ clonePath ++ " && git commit -m\"First commit\""
-        , "cd " ++ clonePath ++ " && git push"
+        , "cd " ++ clonePath ++ " && git push origin master"
```

Comment:
Hm, maybe defaults are different across machines. Since this is just a test repo that we blow away anyway, seems like it should be fine to be explicit.